src/service/resource.js

```diff
@@ -357,10 +365,18 @@ export class Resource {
     const viewport = this.resources_.getViewport();
     this.layoutBox_ = box;
 
+    const win = this.resources_.win;
+    const computedStyle = win./*OK*/getComputedStyle(this.element);
```

Comment:
I'm wondering if we allow `undefined` values, would it be enough for us to just read the values from `element.style`? Calling `getComputedStyle` here for all elements (most of which do not care about resizing) is pretty expensive. If not - I think updating margin is a rare enough case and we should move it closer to the size updating code and only do this when margin changes have been requested.